build/dependencies.yaml

```diff
@@ -18,7 +18,7 @@ dependencies:
 
   # CNI plugins
   - name: "cni"
-    version: 1.2.0
+    version: 1.3.0
```

Comment:
need to include this:

https://github.com/kubernetes/kubernetes/blob/b1e35d5616e066c4b0b3f550c33bbf67bb3f876e/hack/local-up-cluster.sh#L55

Reply:
Fixed and updated.